### DISCUSSION TITLE
Custom widget works fine in dev mode, doesn't load in production

### DISCUSSION THREAD

**Discussion by genox** [medusajs/medusa#5175] [Q&A] [answered]
Hi,

I made an admin widget to edit custom properties. The whole thing works fine locally but when deployed using a docker container, the widget doesn't get included/rendered as I would expect.

- Looks like the API is picking up the changes just fine, I see my custom properties on a product.
- It looks like the build is ok, the transpiled files are in dist/admin/... as expected.

The Dockerfile is standard, more or less: install deps, build, push..

I start the server in production with `cross-env medusa migrations run && medusa start` - but it looks like it is skipping inclusion of the widget.

What's going on?

**Comment by srindom** [comment]
@kasperkristensen do you have an idea for what might be happening here?

**Comment by genox** [comment, marked as answer]
Sorry, after checking the console, I see that my admin widget is throwing an error and the enclosing error boundary is probably preventing it from rendering.

I think it would be good to show the same kind of error boundary output in production since some errors might be caused based on the available data. This would make solving issues like that much more efficient.
